satellite/admin-back-office-v1.md

```diff
@@ -74,42 +74,44 @@ with or without any relation with Storj will be able to use it.
 
 ##### Operations
 
-1. Account:
+1 Account:
```

Comment:
To me, these make more sense as `###### Account` or `<h6>Account</h6>`, because they look more like headers than specific enumeration.

I'm also not quite sure whether numbers are necessary below, these look like a regular bulleted list.

Reply:
I feel that the document has already too many header levels, so I wouldn't like to add one to this list.

I preferred how it was, however, Github doesn't render it properly and I tried several things, between them using `<ol>` tags, but they didn't work either.

When I wrote it, I wanted numbers in case I needed to refer to them in other parts of the document, so I could use for example 1.5.1. In the end, I didn't refer them anywhere (if I'm not wrong), so I could remove all of them.

Do you want me to remove all of them and leave them with just a bullet list?

Reply:
Ah no, it's fine as is.